todo_app/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { comments } from './comments';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
```

Comment:
You've rendered your comments list inside a `header` node, which you probably didn't intend to do.